W-15225622 - RPA Manager release notes 2024-03/1. (#342)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[march-11-2024]]
== March 11, 2024

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID

> | Selecting a process on the *Process Matrix* now filters the selected process. | W-14657931

Reviewer: ```suggestion
| Selecting a process in *Process Matrix* now filters the selected process. | W-14657931
```

> | Now all team assignments of a *Credential* can be removed. | W-14693509

Reviewer: ```suggestion
| You can now remove all team assignments of a *Credential*. | W-14693509
```

> | Email notifications now contain a correct formatted URL. | W-14693359

Reviewer: ```suggestion
| Email notifications no longer contain a malformed URL. | W-14693359
```

| Copying a process does no longer duplicate activity parameters. | W-14999548
| The user workspace now saves changes properly. | W-15147413

> | The list of tag in the *Bot Workflow Library Tags* view will now be loaded and shown. | W-15156528

Reviewer: ```suggestion
| The list of tags in the *Bot Workflow Library Tags* view now shows properly. | W-15156528
```

RPA Manager is the cloud-based control plane for your automation projects.

> [[march-11-2024]]
> == March 11, 2024

Reviewer: ```suggestion
[[march-12-2024]]
== March 12, 2024
```

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID
| Selecting a process in *Process Matrix* now filters the selected process. | W-14657931
| You can now remove all team assignments of a *Credential*. | W-14693509
| Email notifications no longer contain a malformed URL. | W-14693359
| Copying a process does no longer duplicate activity parameters. | W-14999548
| The user workspace now saves changes properly. | W-15147413
| The list of tags in the *Bot Workflow Library Tags* view now shows properly. | W-15156528

> | User can not save public dashboards as private ones. | W-15212411
> | Sometimes click on claim and reject button on second page of team backlog list do not work. | W-15130497

Reviewer: ```suggestion
| Saving public dashboards as private ones no longer fails. | W-15212411
| Claiming or rejecting processes on the second page of the team backlog no longer fails. | W-15130497
```
